app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```diff
@@ -83,7 +83,7 @@
         },
         "CHANNEL_GREETING_TOGGLE": {
           "LABEL": "Enable channel greeting",
-          "HELP_TEXT": "Automatically send a greeting message after the contact's first message in a conversation.",
+          "HELP_TEXT": "Instantly engage customers with automated greeting messages.",
```

Comment:
This sentence is not clear enough to explain what this feature is about. I would re-consider and rewrite the sentence, even if it is wordy. This change feels more like a catchy heading.

Reply:
@Hricha-Shandily Can you take a look?